web/template.py

```diff
@@ -1093,8 +1093,10 @@ def _load_template(self, name):
                 path, cache=self._cache is not None, base=self._base, **self._keywords
             )
         elif kind == "file":
+            with open(path, encoding="utf-8") as tmpl_file:
+                tmpl_str = tmpl_file.read()
             return Template(
-                open(path, encoding="utf-8").read(), filename=path, **self._keywords
+                tmpl_str, filename=path, **self._keywords
```

Comment:
```suggestion
            with open(path, encoding="utf-8") as tmpl_file:
                return Template(tmpl_file.read(), filename=path, **self._keywords)
```

Reply:
Do you really want to place the return-statement in the body of the with-statement?

Reply:
Yes.  This is no problem.  The context manager will close properly even with a return statement.

Reply:
Ok. Applied your suggestion.